### PR TITLE
scientifica: init at 2.1

### PR DESCRIPTION
**Pull request by moni-dz**
[Scientifica](https://github.com/NerdyPepper/scientifica) is a tall, condensed, bitmap font largely based on creep

###### Motivation for this change

Add scientifica font

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/data/fonts/scientifica/default.nix [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{ lib, stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  pname = "scientifica";
+  version = "v2.1";
```
Comment (SuperSandro2000) [inline pkgs/data/fonts/scientifica/default.nix:5]: Versions in nix do not start with the v letter that GitHub suggests to use for versions.

```suggestion
  version = "2.1";
```

File: pkgs/data/fonts/scientifica/default.nix [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  
+  src = fetchurl {
+    url = "https://github.com/NerdyPepper/scientifica/releases/latest/download/${pname}-${version}.tar";
```
Comment (SuperSandro2000) [inline pkgs/data/fonts/scientifica/default.nix:8]:
```suggestion
    url = "https://github.com/NerdyPepper/scientifica/releases/latest/download/${pname}-v${version}.tar";
```

File: pkgs/data/fonts/scientifica/default.nix [hunk 3]
```diff
@@ -0,0 +1,29 @@
+{ lib, stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
```
Comment (SuperSandro2000) [inline pkgs/data/fonts/scientifica/default.nix:3]: Fonts are usually no derivation but just a download with a post phase. You should find some other fonts that do it like this.

File: pkgs/data/fonts/scientifica/default.nix [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+  pname = "scientifica";
+  version = "v2.1";
+
```
Comment (SuperSandro2000) [inline pkgs/data/fonts/scientifica/default.nix:6]:
```suggestion

```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

**Comment by neonfuz** [comment]
@fortuneteller2k Why the change from pname and version to just name, and version in a let block? I think nixpkgs prefers pname, you should change that back.

Also I like this font, what is the status of this, waiting on a re-review?

**Comment by fortuneteller2k** [comment]
> @fortuneteller2k Why the change from pname and version to just name, and version in a let block? I think nixpkgs prefers pname, you should change that back.
> 
> 
> 
> Also I like this font, what is the status of this, waiting on a re-review?

It's waiting on a re-review.

I used `name` because `fetchurl` doesnt let me use `pname` and `version`.

**Comment by neonfuz** [comment]
Oh yeah my bad, I didn't realize the change to fetchurl, makes sense.

**Comment by neonfuz** [comment]
I tested this package, it works well. Added to fonts.fonts in system config and it was usable.

**Pull request merged**